bridge/client/webrtc.js

```diff
@@ -70,10 +70,9 @@
 
             ws = new WebSocket("ws://localhost:10717/bridge");
             ws.onopen = processSendQueue;
-            ws.onmessage = function (evt) {
-                if (_this.onmessage instanceof Function) {
-                    _this.onmessage({"data": atob(evt.data)});
-                }
+            ws.onmessage = function (event) {
+                if (_this.onmessage instanceof Function)
```

Comment:
No good reason other than readability, and instanceof is sufficient here because messageChannel is only used internally.